### PR TITLE
fortest

### PR DESCRIPTION
**Pull request by stephanie1997-kek**

### CODE REVIEW

**Comment by stephanie1997-kek** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -220,3 +220,12 @@ input {
...
 }
+
+/* Courses */
+.reviewsbox {
+  overflow: auto;
+  padding: 10px 0;
+  border-bottom: 1px solid $gray-lighter;
+}
+
+
```
Comment (saobv-2223) [inline app/assets/stylesheets/custom.scss]:
```suggestion
}

```

File: app/views/courses/_course.html.erb [hunk 2]
```diff
@@ -8,10 +8,12 @@
     <%= course.status %>
   </p>
 
-  <%= link_to t(".view"), course, class: "btn btn-primary btn-sm" %>
+  <div style="float:right;">
```
Comment (saobv-2223) [inline app/views/courses/_course.html.erb]: Avoid using inline style, you can write a class or id in css file and call it here

File: app/views/courses/index.html.erb [hunk 3]
```diff
@@ -10,6 +10,8 @@
...
 <%= paginate @course %>
 
-<% if current_user.admin? %>
-  <%= link_to t(".create"), new_course_url, class: "btn btn-primary btn-lg btn-block" %>
-<% end %>
+<div>
+	<% if current_user.admin? %>
+	  <%= link_to t(".create"), new_course_url, class: "btn btn-primary btn-lg btn-block" %>
+	<% end %>
+</div>
```
Comment (saobv-2223) [inline app/views/courses/index.html.erb]: identing only 2 spaces

File: app/views/courses/show.html.erb [hunk 4]
```diff
@@ -28,33 +28,40 @@
...
+            <%= @course.registers.approved.size %>
           </h4>
         </div>
       </div>
     </div>
   </div>
   
-  <div style="padding-left: 59em;">
-    <%= link_to t(".register"), new_course_register_path(@course), class: "btn btn-primary btn-sm" %> |
+  <div style="float:right;">
```
Comment (saobv-2223) [inline app/views/courses/show.html.erb]: Same above

**Comment by stephanie1997-kek** [comment]
ready

**Pull request merged**